### ISSUE TITLE
arm image 

### ISSUE THREAD

**Issue by HerrFrutti** [xavier-hernandez/goaccess-for-nginxproxymanager#13]
add arm support.
User on reddit already build one:
[https://hub.docker.com/repository/docker/justsky/goaccess-for-nginxproxymanager](https://hub.docker.com/repository/docker/justsky/goaccess-for-nginxproxymanager)

**Comment by Just5KY** [comment]
and it works as expected.

**Comment by xavier-hernandez** [comment]
@Just5KY since everything seems to be working as-is. I'm leaning towards finally moving this to the main branch in the next few days would this cause you any problems.

**Comment by Just5KY** [comment]
No issues @xavier-hernandez 
I can just build a newer image using the main branch in the future and tag that as `latest`

But yes, the issue  #7  that DBTech mentioned does exist. I am also seeing Cloudflare IPs in the list.

I also have a couple of feature requests, but I will create another issue for those.
Thank-you for creating this tool, I love it. 

![BabyGrootDancingGIF](https://user-images.githubusercontent.com/71321862/156418975-223316e1-c0e1-4927-867a-a2a36089af5d.gif)

**Comment by xavier-hernandez** [comment]
@Just5KY added a link to your arm version in the readme file

**Comment by xavier-hernandez** [comment]
@Just5KY merged everything into the main branch, updated readme

There will be 2 versions, latest (main branch) and develop (develop branch)

https://hub.docker.com/r/xavierh/goaccess-for-nginxproxymanager/tags

**Comment by Just5KY** [comment]
Nice
I will push the arm image based on this tags

**Comment by Just5KY** [comment]
I am adding armv7 support too.
Using Docker buildx we can create multi-arch images.

`docker buildx build --platform linux/arm/v7,linux/arm64/v8 --push -t justsky/goaccess-for-nginxproxymanager:develop .`

**Comment by Just5KY** [comment]
![image](https://user-images.githubusercontent.com/71321862/156507560-0a06a121-1cba-45df-82ee-f3d858957f30.png)

**Comment by xavier-hernandez** [comment]
> I am adding armv7 support too. Using Docker buildx we can create multi-arch images.
> 
> `docker buildx build --platform linux/arm/v7,linux/arm64/v8 --push -t justsky/goaccess-for-nginxproxymanager:develop .`

Yeah, I was going to set that up but since you got around to it I'll let you have it :) I also don't have any arm devices to test with other than an old raspberry pi.

**Comment by Just5KY** [comment]
Lol 
I am building all the images on my Pi4

**Comment by HerrFrutti** [comment]
Yeah I've run your (@Just5KY ) image since yesterday on my pi4 and it's working.
